### PR TITLE
Load plugins from app dir on Windows

### PR DESCRIPTION
**Pull request by pelijah**
**Test plan (required)**

**Closing issues**

### CODE REVIEW

File: src/plugins/PluginManager.h [hunk 1]
```diff
@@ -21,6 +21,7 @@ Q_OBJECT
      * @brief Load all plugins, should be called once on application start
      */
     void loadPlugins();
+    void loadPlugins(const QDir &pluginsDir);
```
Comment (thestr4ng3r) [inline src/plugins/PluginManager.h]: Please add a bit of documentation to differentiate between the two loadPlugins methods, so it's clear that the one without parameters will call the other with the default paths.

File: src/plugins/PluginManager.cpp [hunk 2]
```diff
@@ -53,9 +53,22 @@ void PluginManager::loadPlugins()
         qCritical() << "Failed to get a path to load plugins from.";
         return;
     }
-    QDir pluginsDir(pluginsDirStr);
 
+    loadPlugins(QDir(pluginsDirStr));
+
+#ifdef Q_OS_WIN
+    QDir localDir;
```
Comment (thestr4ng3r) [inline src/plugins/PluginManager.cpp]: Do we really want to call it "local" dir? For me, it's more like a global dir (on Linux it would be somewhere in /usr/share) and the user-local one would be "more local". It doesn't really matter here since it's just the variable name, but I think we should clarify that at some point.

Comment (xarkes) [reply src/plugins/PluginManager.cpp]: Can be named "appDir" too.
If have a question though, why should it be for Windows only?

Comment (ITAYC0HEN) [reply src/plugins/PluginManager.cpp]: ping @pelijah

Comment (pelijah) [reply src/plugins/PluginManager.cpp]: Does it even make sense for Linux/macOS? On Windows you have exactly one directory and it's much easier to install plugin somewhere to that directory.

Comment (xarkes) [reply src/plugins/PluginManager.cpp]: Well you're right, it's probably not useful for linux or mac os...

File: src/Cutter.pro [hunk 3]
```diff
@@ -49,8 +49,6 @@ equals(CUTTER_BUNDLE_R2_APPBUNDLE, true)        CONFIG += CUTTER_BUNDLE_R2_APPBU
 !defined(CUTTER_APPVEYOR_R2DEC, var)            CUTTER_APPVEYOR_R2DEC=false
 equals(CUTTER_APPVEYOR_R2DEC, true)             CONFIG += CUTTER_APPVEYOR_R2DEC
 
-!defined(CUTTER_APPVEYOR_R2DEC, var)            CUTTER_APPVEYOR_R2DEC=false
```
Comment (xarkes) [inline src/Cutter.pro]: What does this have to do with the current PR?

Comment (pelijah) [reply src/Cutter.pro]: I just removed duped line. See line 49.
